CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v1.1.1](https://github.com/voxpupuli/puppet-network/tree/v1.1.1) (2023-10-14)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-network/compare/v1.1.0...v1.1.1)
+
+**Breaking changes:**
+
+- Drop Puppet 6 support [\#298](https://github.com/voxpupuli/puppet-network/pull/298) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
That should be a major version bump

Reply:
Fixed.  Hopefully that's enough to get it published :)

Reply:
Grrr...the changelog generation seems incomplete.  Can someone please help with that?